linters/eslintrc

```diff
@@ -0,0 +1,31 @@
+env:
```

Comment:
eslint should be --reset and we should define all of the options explicitly as I did [here](https://github.com/ripple/ripple-lib/blob/develop/eslint.json). This way we don't have to look up default eslint rules. eslint also has some unsavory defaults that anyone reviewing this pull request may not be aware of.